lib/server.js

```diff
@@ -731,6 +744,33 @@ WalletService.prototype._selectTxInputs = function(txp, cb) {
   });
 };
 
+
+WalletService.prototype._canCreateTx = function(copayerId, cb) {
+  var self = this;
+  self.storage.fetchLastTxs(self.walletId, copayerId, 5, function(err, txs) {
+    if (err) return cb(err);
+
+    if (!txs.length) 
+      return cb(null, true);
+
+    var lastRejections = _.takeWhile(txs, {status: 'rejected'});
+
+    if (!lastRejections.length) 
+      return cb(null, true);
+
+    var lastTxTs = txs[0].createdOn;
+    var now = Math.floor(Date.now() / 1000);
+    var timeSinceLastRejection = now - lastTxTs;
+    var backoffTime = Math.pow(WalletService.backoffTime,lastRejections.length);
```

Comment:
WalletService.backoffTime is default 2 * 60 = 180 seconds

180^2 = 32400 seconds = 9 hours

1 rejection = 2 minute backoff
2 rejections = 9 hour backoff

Might want to divide by 60 before powering, then multiply by 60 after the powering.

2^2 = 4 then * 60 = 240 seconds